apps/greencheck/templates/backfill_procedure.sql

```diff
@@ -1,14 +1,14 @@
 DROP PROCEDURE IF EXISTS backfill;
 
-CREATE PROCEDURE backfill()
+CREATE PROCEDURE backfill(IN date_begin Date)
 BEGIN
   DECLARE done INT DEFAULT FALSE;
 
   DECLARE gdatum DATETIME;
   DECLARE ggreen TINYINT(2);
   DECLARE gid_hp INT(11);
   DECLARE gurl VARCHAR(255);
-  DECLARE cur CURSOR FOR SELECT datum, green, id_hp, url FROM greencheck where id_hp > 0;
+  DECLARE cur CURSOR FOR SELECT datum, green, id_hp, url FROM greencheck where id_hp > 0 AND datum > date_begin;
```

Comment:
it's a small thing, but can we please have all SQL statements in caps, to more easily differentiate from variables? i.e. WHERE being in caps too?

More here - https://www.sqlstyle.guide/#reserved-words

If possible, for future raw SQL that we write, I'd be happier, where we won't have existing conventions on the project to follow the SQL style outlined in this guide, for readability:

https://www.sqlstyle.guide

Reply:
Good call.